### DISCUSSION TITLE
请问下载4k分辨率视频是否只有30帧的选项？如果是，能否开放4k 60帧的选项？

### DISCUSSION THREAD

**Discussion by sienna911** [the1812/Bilibili-Evolved#3812] [提问 / Q&A] [answered]
举例：BV1jK411U7xf

原视频信息里分辨率为3840 x 2160@62.500，选择dash格式后，清晰度并无提供4k60p的选项，下载到本地后也确实是30帧，能否实现下载4k60p的功能呢？还是有限制无法实现。

**Comment by the1812** [comment, marked as answer]
你给的例子可以下载 60 帧啊
![image](https://user-images.githubusercontent.com/26504152/201342906-8643ccad-bce8-4317-9530-d5af7b41a99f.png)

**Reply by sienna911** [reply]
抱歉，是剪辑软件里设置了30帧，以为是原素材的问题。
